src/middlewared/middlewared/plugins/filesystem_/perm_check.py

```diff
@@ -25,6 +25,62 @@ def check_access(path: str, check_perms: dict) -> bool:
 
 class FilesystemService(Service):
 
+    @private
+    def generate_user_details(self, id_type, xid):
+        if id_type not in ['USER', 'GROUP']:
```

Comment:
We can just perhaps use accepts schema ?

Reply:
Meh, it's a private method, and the args are minimal.